azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/di/DependencyInjectionContainerImpl.kt

```diff
@@ -98,11 +99,14 @@ internal class DependencyInjectionContainerImpl(
     }
 
     override val appStore by lazy {
+        val maxRemoteParticipantLimit = applicationContext.resources.getInteger(R.integer.azure_communication_ui_calling_max_remote_participants)
+
```

Comment:
App store is related to redux state, action management. We should not inject anything related to particiapnts in App Store.